docs/videos/DemoVideo/visualDemo.vtt

```diff
@@ -0,0 +1,94 @@
+WEBVTT
+
+STYLE
+::cue {
+  font-family: 'proxima-nova', sans-serif;

```

Comment:
I'm not sure this would work unless the user had Proxima Nova installed on their system, which is unlikely. If this was in my response to my earlier comment about using that typeface, I was only referring to the deck.

Reply:
Ah didn't realize you were talking about the deck. Removed the styling from the captions.